src/pages/registrationScreen/RegistrationScreen.jsx

```diff
@@ -11,6 +11,12 @@ export function RegistrationScreen() {
   const [mensureUnity, setMensureUnity] = useState(0);
   const [currency, setCurrency] = useState(0);
 
+  const preventMinus = (e) => {
```

Comment:
I don't understand this code, what is he doing ? Because  you just put the prevent default. Do you know tell me what diferente between  arrow function and normal function ?

Reply:
I understand now, the pr title is not good, because the native input already have this feature, you just predicted the event to spread. Take care about your title, I read on my code review

Reply:
only syntax?